.travis.yml

```diff
@@ -23,7 +23,7 @@ before_script:
 #  - gocyclo -over 15 ./multus
 
 script:
-  - ./build
+  - ./build.sh
```

Comment:
In line 36 of .travis.yml, travis packages tgz that is built by 'before_script:' section and put it in github release page and we should change this script too, otherwise tgz file in github is not changed.

Reply:
Okey. Let me clarify...

In [.travis.yml:36-46](https://github.com/intel/multus-cni/blob/dev/network-plumbing-working-group-crd-change/.travis.yml#L36-L46), I've wrote script for travisCI. This script is invoked only tagging as matched '^v[0-9].*$', like v0.2.1. 

This travis script will upload files, that is built at TravisCI's build phase (*1), to github release. So in https://github.com/intel/multus-cni/releases/tag/v2.0,  multus-cni_v2.0_linux_386.tar.gz,  multus-cni_v2.0_linux_amd64.tar.gz and  multus-cni_v2.0_linux_arm.tar.gz are automatically put by TravisCI.

(*1): [.travis.yml:31-41](https://github.com/intel/multus-cni/blob/dev/network-plumbing-working-group-crd-change/.travis.yml#L31-L34) builds multus binary tgz for x86, x86_64 and arm for each commit. This binary is used at above travis upload script and also put to [snapshot](https://bintray.com/redhat-nfvpe/multus-cni-crd-snapshots/snapshot) in bintray.com every time.

Hence I suppose your script, 'release.sh' can be replaced with above script, otherwise we put two binaries into release page (one is from travisCI as I mentioned above and another is from release.sh).

Reply:
@s1061123  @dougbtv: yes, I understood the script now. I will make the changes now to include the `-ldflag` in the `goxc`. I think we can keep the `release.sh` for the user local use, and the travis for the github maintainer ? What you guys think ?

Reply:
@rkamudhan, I agree with you. How about to add some comments in release.sh to mention it (i.e. local use) for users and further developers?

Reply:
ok cool. I will add it now.